packages/core/src/App/Components/Elements/LiveChat/use-livechat.ts

```diff
@@ -1,103 +1,93 @@
+            const domain = /^(.)*deriv\.(com|me|be)$/gi.test(window.location.hostname)
+                ? deriv_urls.DERIV_HOST_NAME
+                : 'binary.sx';
+            const client_information = Cookies.getJSON('client_information', {
+                domain,
+            });
 
-                const { utm_source, utm_medium, utm_campaign } = utm_data || {};
+            const utm_data = Cookies.getJSON('utm_data', { domain });
 
-                const { loginid, email, landing_company_shortcode, currency, residence, first_name, last_name } =
-                    client_information || {};
+            const { utm_source, utm_medium, utm_campaign } = utm_data || {};
 
-                client_first_name = first_name ?? ' ';
-                client_last_name = last_name ?? ' ';
+            const { loginid, email, landing_company_shortcode, currency, residence, first_name, last_name } =
+                client_information || {};
 
-                /* the session variables are sent to CS team dashboard to notify user has logged in
+            client_first_name = first_name ?? ' ';
+            client_last_name = last_name ?? ' ';
+
+            /* the session variables are sent to CS team dashboard to notify user has logged in
                 and also acts as custom variables to trigger targeted engagement */
-                const session_variables = {
-                    is_logged_in: !!is_logged_in,
-                    loginid: loginid ?? ' ',
-                    landing_company_shortcode: landing_company_shortcode ?? ' ',
-                    currency: currency ?? ' ',
-                    residence: residence ?? ' ',
-                    email: email ?? ' ',
-                    utm_source: utm_source ?? ' ',
-                    utm_medium: utm_medium ?? ' ',
-                    utm_campaign: utm_campaign ?? ' ',
-                };
-                window.LiveChatWidget?.call('set_session_variables', session_variables);
+            const session_variables = {
+                is_logged_in: !!is_logged_in,
+                loginid: loginid ?? ' ',
+                landing_company_shortcode: landing_company_shortcode ?? ' ',
+                currency: currency ?? ' ',
+                residence: residence ?? ' ',
+                email: email ?? ' ',
+                utm_source: utm_source ?? ' ',
+                utm_medium: utm_medium ?? ' ',
+                utm_campaign: utm_campaign ?? ' ',
+            };
+            window.LiveChatWidget?.call('set_session_variables', session_variables);
 
-                if (is_logged_in) {
-                    // client logged in
-                    // prepfill name and email
-                    window.LiveChatWidget?.call('set_customer_email', session_variables.email);
-                    window.LiveChatWidget?.call('set_customer_name', `${client_first_name} ${client_last_name}`);
+            if (is_logged_in) {
+                // client logged in
```

Comment:
hey @yauheni-deriv do we need the comments ?

Reply:
hey @Dhruv-deriv , not sure bro. I just left them as they were here before.
I think that the code is quite self-descriptive and we can remove them

Reply:
alrightt 👍